index.html

```diff
@@ -14,6 +18,128 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+    <header id="header">
+        <nav class="flex">
+            <a href="#header">Intro</a>
+            <a href="#history">History</a>
+            <a href="#skills">Skills</a>
+            <a href="#contact">Contact</a>
+            <a href="https://ericbanker12.github.io/portfolio-website/">Portfolio</a>
+        </nav>
+    </header>
+    <section id="intro" class="flex intro">
+        <img src="images/me.jpg" alt="Eric Banker Photograph">
+        <div>
+            <h1>Hello, I am Eric Banker.</h1>
+            <h3>I am an aspiring web and software developer.</h3>
+            <p>Initially, I majored in biology to pursue a career in research. However, I felt frustrated with the career prospects and slow progression, so I chose to instead pursue a career in web and software development.</p>
+        </div>
+    </section>
+    <section id="history" class="history">
+        <h1>Work History and Education</h1>
+        <div class="flex">
+            <!-- <div>
+                <h3>Work History</h3>
+            </div> -->
+            <div>
+                <h3>Education</h3>
+                <ul>
+                    <li>May, 2020 – Full Stack Web Development bootcamp graduate,  Lambda School</li>
+                    <li>December, 2018 - Bachelor of Science, Southern Illinois University Edwardsville
+                        <ul>
+                            <li>Major: Biological Sciences</li>
+                            <li>Major Concentration: Genetics and Cellular Biology</li>
+                            <li>Minor: Chemistry</li>
+                            <li>Graduated <i>Summa Cum Laude</i>: Earned 4.0 GPA</li>
+                            <li>Electives included Bioinformatics, Calculus, Quantitative Methods, and Statistics</li>
+                        </ul>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <section id="skills" class="skills">
+        <h1>Skills</h1>
+        <div class="flex">
+            <div>
+                <h3>Languages, APIs, CLIs</h3>
+                <ul>
+                    <li>Apache</li>
+                    <li>AutoHotkey</li>
+                    <li>CSS
+                        <ul>
+                            <li>Bootstrap</li><li>LESS</li>
+                        </ul>
+                    </li>
+                    <li>DOS/Windows Shell</li>
+                    <li>Git BASH/UNIX Shell</li>
+                    <li>HTML</li>
+                    <li>Java</li>
+                    <li>Javascript</li>
+                        <ul>
+                            <li>Electron</li><li>jQuery</li><li>MongoDB</li><li>NodeJS</li><li>React</li>
```

Comment:
Just a reminder- for readability, it does help to put all of these list items on their own line.